backend/coreapp/views/scratch.py

```diff
@@ -441,8 +452,13 @@ def pr(self, request: Request, pk):
         if scratch.owner != request.profile:
             raise ScratchNotOwnerException()
         # TODO: catch exceptions
-        fn: ProjectFunction = scratch.project_function
-        token = request.profile.user.github.access_token
+        fn: Optional[ProjectFunction] = scratch.project_function
```

Comment:
iirc django doesn't return None when the field is null but actually raises an exception

Reply:
This doesn't seem to be true, at least according to tests